packages/test/test-end-to-end-tests/src/test/gc/gcTrailingOps.spec.ts

```diff
@@ -147,105 +146,84 @@ describeNoCompat("GC trailing ops tests", (getTestObjectProvider) => {
 			assert(blobTimestamp2 !== undefined, `Should have unreferenced blob`);
 		});
 
-		itExpects(
-			`A summary has a datastore and blob unreferenced, but trailing ops referenced them ${
-				tombstoneEnabled ? "after sweep timeout" : "before sweep timeout"
-			}`,
-			tombstoneEnabled
-				? [
-						{
-							eventName:
-								"fluid:telemetry:Summarizer:Running:SweepReadyObject_Revived",
-						},
-						{
-							eventName:
-								"fluid:telemetry:Summarizer:Running:SweepReadyObject_Revived",
-						},
-				  ]
-				: [],
-			async () => {
-				const mainContainer = await provider.makeTestContainer(testContainerConfig);
-				const mainDefaultDataStore = await requestFluidObject<ITestDataObject>(
-					mainContainer,
-					"default",
-				);
-				await waitForContainerConnection(mainContainer);
-
-				// Create a data store and blob.
-				const newDataStore =
-					await mainDefaultDataStore._context.containerRuntime.createDataStore(
-						TestDataObjectType,
-					);
-				assert(newDataStore.entryPoint !== undefined, `Should have a handle`);
-				const blobContents = "Blob contents";
-				const blobHandle = await mainDefaultDataStore._runtime.uploadBlob(
-					stringToBuffer(blobContents, "utf-8"),
-				);
+		it(`A summary has a datastore and blob unreferenced, but trailing ops referenced them ${
+			tombstoneEnabled ? "after sweep timeout" : "before sweep timeout"
+		}`, async () => {
```

Comment:
This an interesting side effect of the change. I am looking into what's happening specifically but the difference here is that the [TestContainerRuntimeFactory](https://github.com/microsoft/FluidFramework/blob/1837158bc59425dc0e4af6aa463e3b5f238b0e0e/packages/test/test-utils/src/testContainerRuntimeFactory.ts#L55) now calls `getAliasedDataStore` during initialization which does an extra step of getting the entry point which loads the DDSs as well. So, trailing ops are processed by DDS **_before_** container is connected and garbage collector initializes and does not generate these event.s